.pre-commit-config.yaml

```diff
@@ -3,19 +3,19 @@ repos:
     hooks:
       - id: web-js-lint-check
         name: Web JS Lint Check
-        entry: sh ./pre-commit/web_lint_check.sh js
+        entry: python ./pre-commit/web_lint_check.py js
```

Comment:
This may not behave as expected on some Linux platforms which don't have a python or link python to python2; just python3.

Ideally, set the execute bit on the script in git and use the regular `#!/usr/bin/env python3` thing, and just leave the interpreter off of these calls.

Reply:
I thought all of us installed Python, so I assumed we all have `python` or `python3` command.
We need to handle if which command is available either  `python` or `python3`, but i think we dont need to use `#!/usr/bin/env python3`?

Reply:
python3 is encouraged by [PEP-394](https://peps.python.org/pep-0394/), so should be safe to rely upon.  I would've put a bangpath in the executable script and just directly called it, but I'm definitely not going to die on that hill. :)

Reply:
Since we run make in a virtualenv it is safe to assume that python is indeed python3 but I agree that scripts should generally use `#!` and `python3` is preferable.

Reply:
Ok i'll change it

Reply:
Yeah, several "long term support" platforms (SUSE enterprise / OpenSUSE Leap, Ubuntu though at least 20.04, I think RHEL) which are still in use leave python linked to python2 to support old python2 admin tools.